Dockerfile-mongo-seed

```diff
@@ -0,0 +1,13 @@
+FROM  mongo:3.2
+
+COPY dc-campaign-finance-mongodatabase.zip .
+
+# update packages and intall unzip
+RUN apt-get update
```

Comment:
This and the next line (and the third if you use the apt-get remove) can be done in one single block, which will create only one layer in the Docker image. Something like:

```
RUN apt-get update \
    && apt-get install unzip \
    && apt-get remove --purge -y unzip \
    && rm -rf /var/lib/apt/lists/*
```

This helps reduce the size of your Docker image.